eyieko/landville-frontend
# #167416901 authenticated users can view profile

## Description ##
- Enable user to access profile route is authenticated.

## Type of change ##
Please select the relevant option

## How to test this PR
- Click on the login button and enter your details to be authenticated.
- Click on the profile icon that can be found on the navbar, and a dropdown will appear. 
- Click on profile and you should be able to view your profile.

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes

## Checklist: ##

## Related Pivotal Tracker stories ##
[#167416901](https://www.pivotaltracker.com/story/show/167416901)
